### DISCUSSION TITLE
Future of fast-foundation

### DISCUSSION THREAD

**Discussion by fcollonval** [Adaptive-Web-Community/Adaptive-Web-Components#208] [General]
For what I understand from the new plan for the FAST repository, fast-foundation v3 will be released and then immediately archived as only fast-element will be maintained.

If this is correct, would an "official" fork be maintained by this project?

**Comment by Dovrol** [comment]
also curious about this one

**Comment by bheston** [comment]
Somehow I didn't see a notification when this came in, but that is what we had planned to do. There are some long-term considerations about how to move forward, ideally supporting more foundational libraries with Adaptive UI, but this is a good start.

I've been pretty busy on the Adaptive UI side and am not using the web components much these days. I hope this is a pretty simple move. I can try to get to it soon, or let me know if you are still interested in helping.
